content/en/docs/release-notes/rn-armory-operator/armory-operator-v1-1-0.md

```diff
@@ -0,0 +1,30 @@
+---
+title: v1.1.0 Armory Operator
+toc_hide: true
+---
+
+## 08/11/2020 Release Notes
+
+## Known Issues
+No known issues.
+
+## Armory Operator
+
+* chore(release): upgrade to oss operator 1.1.0
+* fix(kustomize): Make kustomization.yml more compatible
+* chore(vault): Error messages not capitalized
+* feat(vault): Mount vault's self-signed CA into services
+
+## Spinnaker Operator
+
+* fix(validation): name ports in generated service, declare container ports in manifest
+* fix(accounts): Add Spring profile on specialized services
+* feat(expose/ingress): ingress support, name all ports on services, fix config clone
+* fix(validation): Issue patch to update status
+* fix(validation): Add parallel validation support
```

Comment:
Perhaps a note that validations are now executed in parallel and should therefore finish faster.